Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+PathComponent.swift

```diff
@@ -13,7 +13,16 @@ import SymbolKit
 /// All known symbol kind identifiers.
 ///
 /// This is used to identify parsed path components as kind information.
-private let knownSymbolKinds = Set(SymbolGraph.Symbol.KindIdentifier.allCases.map { $0.identifier })
+private let knownSymbolKinds = Set<String>(
+    (SymbolGraph.Symbol.KindIdentifier.allCases + [
+        .extendedProtocol,
```

Comment:
Just double-checking: This is a non-breaking change for authored links, right? Because the existing links that are disambiguated with a USR hash will continue to resolve?

Reply:
Yes. Previously if an author had written `Foo-struct.extension` we would look for a symbol named  `Foo-struct.extension` and fail. With this change we'll look for a symbol named `Foo` that is a kind of `struct.extension` instead.